Makefile

```
@@ -1,9 +1,12 @@
 SHELL = /bin/bash
 COMMIT = $(shell git rev-parse --short=7 HEAD)$(shell [[ $$(git status --porcelain) = "" ]] || echo -dirty)
 ARO_IMAGE ?= ${RP_IMAGE_ACR}.azurecr.io/aro:$(COMMIT)
+AUTOREST_IMAGE_REPO = quay.io/openshift-on-azure/autorest
 
 # fluentbit version must also be updated in RP code, see pkg/util/version/const.go
 FLUENTBIT_VERSION = 1.7.8-1
+AUTOREST_VERSION = 3.3.2
+AUTOREST_IMAGE = "${AUTOREST_IMAGE_REPO}:${AUTOREST_VERSION}"
```

Since we don't use `AUTOREST_IMAGE_REPO`  elsewhere, we can probably just include it in this line and remove the var:
```suggestion
AUTOREST_IMAGE = "quay.io/openshift-on-azure/autorest:${AUTOREST_VERSION}"
```